js-basics/js-variables-and-numbers_simple-calculator/js/index.js

```diff
@@ -15,26 +15,38 @@ const moduloButton = document.querySelector('[data-js="modulo"]');
 
 addButton.addEventListener("click", () => {
   // 1. Add the two operands and store the result in a variable. Log the variable's value to the console.
+  let add = operand1 + operand2
```

Comment:
always default to `const` if you're not changing the variable

Reply:
fixed